cmux-tui/frontends/web/src/styles.css

```
@@ -132,6 +133,7 @@ button { color: inherit; touch-action: manipulation; }
 .pane-rule { flex: 1 1 auto; min-width: 0; height: var(--tui-cell-height); border-top: 1px solid currentColor; transform: translateY(calc(var(--tui-cell-height) / 2)); }
 .pane-clients-trigger { flex: 0 0 auto; height: var(--tui-cell-height); padding: 0 calc(var(--tui-cell-width) / 2); border: 0; background: var(--terminal-background); color: currentColor; font: inherit; line-height: var(--tui-cell-height); white-space: nowrap; cursor: pointer; }
 .pane-clients-trigger:hover, .pane-clients-trigger[aria-expanded="true"] { color: var(--status-active-fg); background: var(--status-active-bg); }
+.pane-clients-label { flex: 0 0 auto; height: var(--tui-cell-height); padding: 0 calc(var(--tui-cell-width) / 2); color: currentColor; line-height: var(--tui-cell-height); white-space: nowrap; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)